src/Psalm/Internal/Type/SimpleAssertionReconciler.php

```diff
@@ -2501,10 +2516,55 @@ function (Type\Union $t): bool {
             return $existing_var_type;
         }
 
-        $failed_reconciliation = 2;
+        $failed_reconciliation = Reconciler::RECONCILIATION_EMPTY;
 
         return $assertion === 'empty' && $existing_var_type->possibly_undefined
             ? Type::getEmpty()
             : Type::getMixed();
     }
+
+    /**
+     * @param   0|1|2    $failed_reconciliation
```

Comment:
while we're on it, care to replace those phpdoc with Reconciler::RECONCILIATION_* everywhere?

Reply:
I tried that but psalm failed when doing so. Thats why I've removed it again as I am quite busy ATM and don't want to spent more time on this than necessary.

There are plenty of issues with referencing class-constants using wildcards but I cannot fix them all.

Reply:
No worries then, I'll do it later ;)